.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+## Context
+
+[provide introduction to the issue , why it is relevant, and who it impacts]
+
+## Process
+
+[ordered list the process to finding and recreating the issue, example below]
+
+1. Login to site as a Site Owner
```

Comment:
To save time and space in titles, I think we should standardize on SO, CE, EO, ADMIN and DEV for roles.